.github/workflows/build.yaml

```
@@ -65,22 +63,34 @@ jobs:
               - 'dataplane/**'
             containerfile:
               - 'build/Containerfile.dataplane'
-      - name: Setup QEMU
-        if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
-        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
-      - name: Setup Docker Buildx
+      - name: Build container image for amd64
         if: steps.filter.outputs.dataplane || steps.filter.outputs.containerfile
-        id: buildx
-        uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
+        uses: docker/build-push-action@5cd11c3a4ced054e52742c5fd54dca954e0edd85 # v6.7.0
```

From what I can tell from https://github.com/docker/build-push-action/commit/5cd11c3a4ced054e52742c5fd54dca954e0edd85, I think this is actually `v6.15.0`?

```suggestion
        uses: docker/build-push-action@5cd11c3a4ced054e52742c5fd54dca954e0edd85 # v6.15.0
```